pipeline/detection.py

```diff
@@ -74,7 +167,16 @@ def run(self, *args, **kwargs):
                         f'Cannot find a subtraction image corresponding to the datastore inputs: {ds.get_inputs()}'
                     )
 
-                detections = self.extract_sources(image)
+                # TODO -- should probably pass **kwargs along to extract_sources
+                #  in any event, need a way of passing parameters
+                if self.pars.method == 'sep':
+                    detections = self.extract_sources_sep(image)
+                elif self.pars.method == 'sextractor':
+                    psffile = None if self.pars.psf is None else self.pars.psf.get_fullpath()
+                    detections, _ = self.extract_sources_sextractor( image, psffile=FIX_THIS )
```

Comment:
Not sure what is going on. Why not just put `psffile` into the function?

Reply:
....weird.  I don't know why the tests passed, there should have been an error about a file not found.

...AH.  Yeah. I see.  I need to take that parameter in extract_sources_sextractor, and actually use it.  It didn't thrown an error because it was just hidden and ignored in **kwargs

Reply:
(This is what happens when you realize that a meeting is starting when you're right in the middle of typing a line of code.)